src/explore-education-statistics-frontend/src/modules/find-statistics/PublicationReleasePage.tsx

```diff
@@ -141,7 +141,7 @@ const PublicationReleasePage: NextPage<Props> = ({ data }) => {
             >
               <ul className="govuk-list govuk-list--bullet">
                 {data.downloadFiles.map(({ extension, name, path, size }) => (
-                  <li key={path}>
+                  <li key={path} className="dfe-align--left">
```

Comment:
This won't fix this problem as it's in the admin's content preview, not the public. To make it happen, you just need a sufficiently long subject name I think.